src/assets/styles/components/cocktail-box.scss

```diff
@@ -6,9 +6,15 @@
   min-height: 200px;
 
   h3 {
+    color: $black;
```

Comment:
is this still explicitly needed after having `color` defined on the body?

Reply:
Yes, colors are getting inherited from a

Reply:
i will look into that